@uportal/eyebrow-user-info/vue.config.js

```diff
@@ -12,14 +12,5 @@ module.exports = {
     i18n: {
       enableInSFC: true
     }
-  },
-  css: {
-    loaderOptions: {
-      css: {
-        modules: true,
-        localIdentName: '[name]-[local]',
-        camelCase: 'only'
-      }
-    }
```

Comment:
@jgribonvald css modules don't appear to be used in this package.
would it be okay to disable this plugin?

If not that's okay, it will take some additional migration work to get it to work with Vue CLI v4 though
https://cli.vuejs.org/migrating-from-v3/#vue-cli-service

Reply:
it looks like it is required, added 56ece446b537ec643877ca2d505bba2dd22c8080 to migrate to the new loader syntax